pkg/scheduler/plugins/sla/sla.go

```diff
@@ -124,12 +124,15 @@ func (sp *slaPlugin) OnSessionOpen(ssn *framework.Session) {
 
 		lCreationTimestamp := lv.CreationTimestamp
 		rCreationTimestamp := rv.CreationTimestamp
-		if lCreationTimestamp.Add(*lJobWaitingTime).Before(rCreationTimestamp.Add(*rJobWaitingTime)) {
+
```

Comment:
I think we can use time's `Compare` method to replace the statement, which is more concise.

Reply:
please show more info. thanks so much. @Monokaix

Reply:
We can call the method `time.Compare` to replace if else check.